### PR TITLE
typetests cleanup

### PR DESCRIPTION
**Pull request by sonalivdeshpande**

### CODE REVIEW

File: packages/dds/map/package.json [hunk 1]
```diff
@@ -96,6 +96,43 @@
   },
   "typeValidation": {
     "version": "2.0.0",
-    "broken": {}
+    "broken": {
```
Comment (ssimic2) [inline packages/dds/map/package.json:99]: Can you check why these tests are broken? I don't see breaking changes in 1.2.0: https://github.com/microsoft/FluidFramework/pull/11025/files#diff-ca95ce2077e008bf603122e069ee1da158095cf74681aa5cac14657d83a97376

File: packages/framework/fluid-framework/package.json [hunk 2]
```diff
@@ -105,6 +105,38 @@
       },
       "EnumDeclaration_ConnectionState": {
         "forwardCompat": false
+      },
```
Comment (ssimic2) [inline packages/framework/fluid-framework/package.json:108]: Similar here, please check to see if this was expected breaking change.

File: packages/dds/map/package.json [hunk 3]
```diff
@@ -96,6 +96,43 @@
...
+      },
+      "RemovedTypeAliasDeclaration_IDirectorySubDirectoryOperation": {
+        "forwardCompat": false,
+        "backCompat": false
+      },
+      "RemovedInterfaceDeclaration_ILocalValue": {
+        "forwardCompat": false,
+        "backCompat": false
+      }
+    }
```
Comment (ssimic2) [inline packages/dds/map/package.json:136]: Could it be related to this: https://github.com/microsoft/FluidFramework/pull/11011/files ? I'm not sure if this was released as 1.2

Comment (sonalivdeshpande) [reply packages/dds/map/package.json:136]: seems to be related. it is part of 1.2 release

File: packages/framework/fluid-framework/package.json (hunk 2 above)
Comment (sonalivdeshpande) [reply packages/framework/fluid-framework/package.json:108]: these seem to be in next already: https://github.com/microsoft/FluidFramework/blob/next/packages/framework/fluid-framework/package.json

File: packages/dds/map/package.json (hunk 3 above)
Comment (ssimic2) [reply packages/dds/map/package.json:136]: @Josmithr was this a breaking change on main? Did you need to fix typetests?

Comment (Josmithr) [reply packages/dds/map/package.json:136]: That PR strictly added items to the API surface that were already transitively included. It should have been strictly non-breaking.

File: packages/framework/fluid-framework/package.json (hunk 2 above)
Comment (ssimic2) [reply packages/framework/fluid-framework/package.json:108]: This is different set. Related to Josh's PR.

**Review by ssimic2 — APPROVED** [review]

**Comment by sonalivdeshpande** [comment]
closing this PR in favor of https://github.com/microsoft/FluidFramework/pull/11043

**Pull request closed without merging**